db/migrate/20220429084831_add_body_slot_to_canonical_clothing_items.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-class AddBodySlotToCanonicalClothingItems < ActiveRecord::Migration[6.1]
```

Comment:
I reconsidered this migration since it was easy to add the body_slot column to the original migration creating the table.